src/utils.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013, Fredrik Hultin.
+// Copyright 2013, Jakob Bornecrantz.
+// SPDX-License-Identifier: BSL-1.0
+/*
+ * OpenHMD - Free and Open Source API and drivers for immersive technology.
+ */
+
+/* Utility functions */
+
+#ifndef UTILS_H
+#define UTILS_H
+
+#include <wchar.h>
+
+static int ohmd_wstring_match(const wchar_t *a, const wchar_t *b)
```

Comment:
Making this `static inline` will silence a bunch of unused function warnings:
> ../src/utils.h:15:12: warning: ‘ohmd_wstring_match’ defined but not used [-Wunused-function]

Reply:
ok done